mobile/native/src/state.rs

```diff
@@ -85,8 +86,6 @@ pub fn try_get_storage() -> Option<TenTenOneNodeStorage> {
 /// Lazily creates a multi threaded runtime with the the number of worker threads corresponding to
 /// the number of available cores.
 pub fn get_or_create_tokio_runtime() -> Result<&'static Runtime> {
-    static RUNTIME: Storage<Runtime> = Storage::new();
```

Comment:
It looked weird to me as it would create a new runtime everytime we calling this function? I am not sure about the rust syntax here. It might also be fine. I just aligned it with the other static storages that we keep in that file.